op-node/rollup/derive/channel_bank.go

```diff
@@ -124,6 +124,7 @@ func (cb *ChannelBank) Read() (data []byte, err error) {
 	if !ch.IsReady() {
 		return nil, io.EOF
 	}
+	cb.log.Info("Reading channel", "channel", first, "frames", len(ch.inputs))
```

Comment:
Should we change these logs to debug? IMHO it's a bit too verbose for the regular user.

Reply:
The open log is info. Adding individual frames is debug.